_posts/2020-04-23-Wk5-Q+A-blog.md

```diff
@@ -0,0 +1,46 @@
+---
+Author: John
+Title: “Open Marketplace:  Local, Feeless, and Open-Source!”
```

Comment:
Could you check to see why your computer is using special quotation characters instead of plain ones (e.g. `"`)?  I think it's causing the title to not get picked up in the blog [listing](https://deploy-preview-102--codevid-19-staging.netlify.app/blog) or the [article](https://deploy-preview-102--codevid-19-staging.netlify.app/2020/04/23/2020-04-23-wk5-q+a-blog) itself:

<img width="1081" alt="image" src="https://user-images.githubusercontent.com/6334517/80161116-94684b00-858c-11ea-805b-453429435c54.png">
<img width="1465" alt="image" src="https://user-images.githubusercontent.com/6334517/80161191-cc6f8e00-858c-11ea-8fef-699c765c0a2b.png">

Reply:
absolutely-I was pasting from a google doc-and there was something up with my ' character recently as well.  Would it be keyboard language?  us v canada?

Reply:
fixed